site/content/docs/developing/secrets.en.md

```diff
@@ -49,13 +49,18 @@ Suppose you have a Secrets Manager secret with the following configuration:
 You can modify your manifest file with:
 ```yaml
 secrets:
-  # Option 1. Referring to the secret by name.
+  # Option 1. Referring to the secret by name, if your secret name does not contain a hyphen (e.g. mysql). If it does (e.g. mysql-db), see Option 2.
   DB:
     secretsmanager: 'mysql'
   # You can refer to a specific key in the JSON blob.
   DB_PASSWORD:
     secretsmanager: 'mysql:password::'
 
-  # Option 2. Alternatively, you can refer to the secret by ARN.
+  # Option 2. Refering to the secret by name, with the random 6-character suffix.
+  # If the secret name contains a hyphen (e.g. mysql-db instead of mysql), then you have to include the 6-character suffix. Otherwise, secretsmanager won't be able to find your secret.
```

Comment:
Wut any doc we can refer to for this random fact 🤯

Reply:
YES! The random fax: https://docs.aws.amazon.com/secretsmanager/latest/userguide/troubleshoot.html#ARN_secretnamehyphen

Reply:
Gotcha! Maybe add the link to the doc just for reference?